src/Core/Content/Sitemap/SitemapException.php

```diff
@@ -20,4 +25,26 @@ public static function fileNotReadable(string $path): self
             ['path' => $path]
         );
     }
+
+    public static function sitemapAlreadyLocked(SalesChannelContext $context): self
+    {
+        return new self(
+            Response::HTTP_BAD_REQUEST,
+            self::SITEMAP_ALREADY_LOCKED,
+            'Cannot acquire lock for sales channel {{salesChannelId}} and language {{languageId}}',
+            [
+                'salesChannelId' => $context->getSalesChannelId(),
+                'languageId' => $context->getLanguageId(),
+            ],
+        );
+    }
+
+    public static function invalidDomain(): self
+    {
+        return new self(
+            Response::HTTP_BAD_REQUEST,
+            self::INVALID_DOMAIN,
```

Comment:
Can we get more precise here? Or would you say you know immediately what happend if you get that messages?